docs/Measuring_Resonances.md

```diff
@@ -31,9 +50,31 @@ and **will not work**. The recommended connection scheme:
 | SDA | 19 | GPIO10 (SPI0_MOSI) |
 | SCL | 23 | GPIO11 (SPI0_SCLK) |
 
-An alternative to the ADXL345 is the MPU-9250 (or MPU-6050).  This
-accelerometer has been tested to work over I2C on the RPi at 400kbaud.
-Recommended connection scheme for I2C:
+Fritzing wiring diagrams for some of the ADXL345 boards:
+
+![ADXL345-Rpi](img/adxl345-fritzing.png)
+
+#### I2C Accelerometers
+
+Suggested twisted pair order:
+
+```
+GND+SDO
+3.3V+SDA
+GND+SCL
+```
+
+If you still experience signal integrity issues and can't shorten your cable, replacing the pull up resistors to lower resistance on SDA and SCL may help. [This document](https://www.ti.com/lit/an/slva689/slva689.pdf) explained the calculation behind choosing the right pull up resistor value. **This may cause DAMAGE to the controller/RPI and Accelerometer!!!**
+
+##### MPU-9250/MPU-6050/MPU-6500
+
+**MPU-6050 and MPU-9250 are obsolete parts! MPU-6500 are replacements of MPU-6050 at a lower failure rate.**
```

Comment:
> MPU-6050 and MPU-9250 are obsolete parts!

The exclamation mark is probably too much here, it's not like it is something to pay super close attention to.

Reply:
I would probably suggest just removing the whole last line.  I haven't seen any evidence of problems with MPU-6050 (and in fact some searches suggest quite the opposite - that they are still used in many projects working for years!).  imo, I think we should leave it to the reader to have done their research on the parts and pick the one they like.

BTW - even the MPU-6500 is not currently recommended for new designs.  It seems invensense is moving to the ICM-* products to replace (if curious you can see a [fork with tested support for the ICM-20948 here](https://github.com/bluesforte/klipper-mpu9250/tree/mpu-blues-icm)).